### PR TITLE
feat : Snackbar/Toast component Added

### PR DESCRIPTION
**Pull request by vikrantkode**
# Snackbar / Toast Added
### `JS` part not included only `HTML` and `CSS`
Screenshot (110)

### CODE REVIEW

**Review by harshitpaliwal95 — CHANGES_REQUESTED** [review]
set Snackbar to normal like your other demo components
then if you can implement functionality for demo Snackbar

**Comment by harshitpaliwal95** [comment]
over all good to go

File: Components/Snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    cursor: pointer;
+}
+
+.icon{
+    cursor: pointer;
+}
+
+.icon:hover{
+    border-radius: 50%;
+    background-color: rgb(74, 74, 77);
```
Comment (sakshi006) [inline Components/Snackbar/snackbar.css:32]: try to use css variables for colors

**Review by sakshi006 — COMMENTED** [review]
Good to go!

File: Components/Snackbar/snackbar.css (hunk 1 above)
Comment (vikrantkode) [reply Components/Snackbar/snackbar.css:32]: yeah in the end i will generalize all the colors from the :root

**Pull request merged**